terraform/aws/security/aws-codebuild-project-unencrypted.tf

```diff
@@ -0,0 +1,58 @@
+# pass
+
+resource "aws_codebuild_project" "enabled" {
+  name         = "example"
+  service_role = "aws_iam_role.example.arn"
+
+  encryption_key = "aws_kms_key.scanner_key.id"
+
+  artifacts {
+    type = "S3"
+  }
+  environment {
+    compute_type = "BUILD_GENERAL1_SMALL"
+    image        = "docker:dind"
+    type         = "LINUX_CONTAINER"
+  }
+  source {
+    type = "NO_SOURCE"
+  }
+}
+
+# fail
+# ruleid: aws-codebuild-project-unencrypted
+resource "aws_codebuild_project" "default" {
+  name         = "example"
+  service_role = "aws_iam_role.example.arn"
+
+  artifacts {
+    type = "S3"
+  }
+  environment {
+    compute_type = "BUILD_GENERAL1_SMALL"
+    image        = "docker:dind"
+    type         = "LINUX_CONTAINER"
+  }
+  source {
+    type = "NO_SOURCE"
+  }
+}
+
+# unknown
+
+resource "aws_codebuild_project" "no_artifacts" {
+  name         = "example"
+  service_role = "aws_iam_role.example.arn"
+
+  artifacts {
+    type = "NO_ARTIFACTS"
```

Comment:
Do you have a preference for whether this case with NO_ARTIFACTS should be flagged? Currently it is flagged, and in that case it'll need a test annotation

Reply:
It may cause false positives (which is valid for all unknown cases). Do you think it's worth increasing coverage for the sake of having more false positives?

Reply:
I think it depends on whether this is an issue worth reporting or not. However, if we're uncertain, I'd say have more false positives -- if it annoys someone more knowledgeable than us, they will make an issue about it ;)